src/apps/lwaftr/ndp.lua

```diff
@@ -2,10 +2,9 @@ module(..., package.seeall)
 
 -- NDP address resolution.
 -- Given a remote IPv6 address, try to find out its MAC address.
--- Exit if this fails.
 -- If resolution succeeds:
 -- All packets coming through the 'south' interface (ie, via the network card)
--- are silently forwarded.
+-- are silently forwarded (unless dropped by the network card).
```

Comment:
If they are dropped by the network card, are they really "coming through the 'south' interface" ? ;)